libs/application/templates/estate/src/lib/dataSchema.ts

```diff
@@ -3,22 +3,40 @@ import { m } from './messages'
 import { parsePhoneNumberFromString } from 'libphonenumber-js'
 import { customZodError } from './utils/customZodError'
 import { EstateTypes, YES, NO } from './constants'
-import * as kennitala from 'kennitala'
 
 const isValidPhoneNumber = (phoneNumber: string) => {
   const phone = parsePhoneNumberFromString(phoneNumber, 'IS')
   return phone && phone.isValid()
 }
 
+const checkIfFilledOut = (arr: Array<string | undefined>) => {
+  if (arr.every((v) => v === '')) {
+    return true
+  } else if (arr.every((v) => v !== '')) {
+    return true
+  } else {
+    return false
+  }
+}
+
```

Comment:
Can be made more concise if you want :)
```suggestion
const checkIfFilledOut = (arr: Array<string | undefined>) => {
  if (arr.every((v) => v === '')) {
    return true
  } else if (arr.every((v) => v !== '')) {
    return true
  }
  return false
}

```